index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Missouri Trans Help</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="http://fonts.googleapis.com/css?family=Annie Use Your Telescope:400,700"
+      rel="stylesheet"
+    />
+    <link
+      href="http://fonts.googleapis.com/css?family=Special Elite:400,700"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://pyscript.net/latest/pyscript.css" />
+    <script defer src="https://pyscript.net/latest/pyscript.js"></script>
+  </head>
+  <body class="page-layout">
+    <h1 class="title">Want to help out queer and trans folk in Missouri?</h1>
+    <p>
+      Take some time to help out our trans friends in Missouri by generating a
+      submission below and copying the details into the
+      <a
+        href="https://ago.mo.gov/file-a-complaint/transgender-center-concerns"
+        target="_blank"
+      >
+        Missoury Transgender Center Concerns Form
+      </a>
+    </p>
+    <div class="python">
+      <py-config>
+        packages = ["random_address","names"]
+        [[fetch]]
+        files = ["complaints_generator.py","data_sets/mo_hospitals.py","data_sets/mo_schools.py","data_sets/ncte_mo_state_report.py","data_sets/states_of_denial.py","data_sets/__init__.py","Person.py"]
```

Comment:
Can you use wildcard syntax here? 

Would it make your life easier if we condensed all the python scripts into a `complaints_generator/` folder, so we didn't have to keep this line up-to-date as we refactor and subdivide the generator code?

Reply:
I actually am uncertain if py-script lets you use wild card syntax, their documentation is a bit rough. I have no background in python, I was mostly trying to get rid of errors until I could at least see something. It would definitely make our lives much easier if we could use wildcard syntax. https://docs.pyscript.net/latest/reference/elements/py-config.html makes it looks like there might be some options to make life slightly easier, but I am not sure exactly what they're doing with the python part so I am unsure

Reply:
Interesting, it looks like this is really just appending things to the python path so the interpreter can properly identify them.

I think creating a root directory is a good idea anyways, and then we can use the `from` keyword to catch everything in that directory without needing to edit this every time we add a new file. I'll put a PR up!